### DISCUSSION TITLE
Is there any way to disable or hide the timer?

### DISCUSSION THREAD

**Discussion by NoxEther** [vkbo/novelWriter#2146] [Help] [answered]
Hi!

The title says it all. This isn't a metric I find useful to track, and having my time spent on a given document timed in real-life under my eyes at the bottom corner of the screen stresses me out, which doesn't help with the creative process. I looked into fiddling with the GUI theme but it doesn't look like that was the way. Thank you in advance!

**Comment by vkbo** [comment]
Hi, there is no setting for it, but you can make a feature request for it under "Issues".

**Reply by HeyMyian** [reply]
There is a feature request here: #1029

**Reply by vkbo** [reply]
Thanks. I seemed to remember this had come up before.

**Reply by NoxEther** [reply]
Thank you both!

**Reply by vkbo** [reply, marked as answer]
I've resolved this in PR #2149. It will be available in 2.6. You can hide the timer by clicking it in the status bar. It will only hide the clock part of it, and just leave the icon. If you click the icon, the timer will show again. The visibility state should persist between writing sessions.

**Reply by NoxEther** [reply]
That was so fast! Thank you so much, kind developer!! I will be looking forward to 2.6, then :D
